Code/Editor/LevelFileDialog.cpp

```diff
@@ -432,6 +438,12 @@ bool CLevelFileDialog::ValidateSaveLevelPath(QString& errorMessage) const
         return false;
     }
 
+    if (!ui->nameLineEdit->hasAcceptableInput())
+    {
+        errorMessage = tr("The level name contains illegal characters.");
```

Comment:
Can we include the level name in the error message? and potentially what the illegal character was, sometimes it can be hard to tell from the ui.

Reply:
Yes, although at the moment this message never gets displayed, I'll refactor it so it does.